tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -301,6 +305,199 @@ def run_conv_with_split(
     assert_with_pcc(torch_output_tensor, torch_out_golden_tensor, pcc=pcc)
 
 
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 16384}], indirect=True)
+@pytest.mark.parametrize("stride", [1, 2])
+@pytest.mark.parametrize(
+    "output_channels, input_channels, input_height, input_width, filter_height, filter_width, pad_h, pad_w, act_block_w_div",
+    (
+        (128, 128, 8, 8, 3, 3, 1, 1, 1),
+        (128, 256, 8, 8, 3, 3, 1, 1, 1),
+        (128, 256, 8, 8, 3, 3, 1, 1, 1),
+        (128, 256, 8, 8, 3, 3, 1, 1, 1),
+        (256, 256, 8, 8, 3, 3, 1, 1, 1),
+        (256, 2048, 8, 8, 3, 3, 1, 1, 8),
+        (512, 2048, 8, 8, 3, 3, 1, 1, 4),
+        (512, 2048, 16, 16, 3, 3, 1, 1, 4),
+        (768, 768, 16, 16, 3, 3, 1, 1, 1),
+        (768, 768, 8, 8, 3, 3, 1, 1, 1),
+        (768, 768, 10, 10, 3, 3, 0, 0, 1),
+        (768, 768, 16, 16, 3, 3, 1, 1, 1),
+        (1280, 1280, 16, 16, 3, 3, 1, 1, 1),
+        (1280, 1280, 8, 8, 3, 3, 1, 1, 1),
+        (1280, 1280, 8, 8, 3, 3, 1, 1, 1),
+        (1280, 2560, 16, 16, 3, 3, 1, 1, 2),
+    ),
+)
+@pytest.mark.parametrize(
+    "has_bias",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "weights_dtype",
+    [ttnn.bfloat8_b, ttnn.bfloat16],
+)
+@pytest.mark.parametrize(
+    "activations_dtype",
+    [ttnn.bfloat8_b, ttnn.bfloat16],
+)
+def test_conv_ws(
+    device,
+    use_program_cache,
+    output_channels,
+    input_channels,
+    input_height,
+    input_width,
+    filter_height,
+    filter_width,
+    pad_h,
+    pad_w,
+    act_block_w_div,
+    stride,
+    has_bias,
+    weights_dtype,
+    activations_dtype,
+):
+    if is_grayskull():
+        if input_channels >= 2048:
+            pytest.skip("Skipping on grayskull due to insufficient L1")
+        if input_channels >= 768 and input_height >= 10:
+            pytest.skip("Skipping on grayskull due to insufficient L1")
+
+    stride_h = stride
+    stride_w = stride
+    batch_size = 2
+    fp32_accum = False
+    packer_l1_acc = False
+    deallocate_activation = False
+    debug = False
+    groups = 1
+
+    torch.manual_seed(0)
+    conv_input_shape = [batch_size, input_channels, input_height, input_width]
+    conv_weight_shape = [output_channels, input_channels // groups, filter_height, filter_width]
+    conv_bias_shape = [1, 1, 1, output_channels]
+    torch_input_tensor_nchw = torch.randn(conv_input_shape, dtype=torch.bfloat16).float()
+    # torch_input_tensor_nchw = torch.ones(conv_input_shape, dtype=torch.bfloat16).float()
```

Comment:
why leave so much commented code? it should be removed

Reply:
Removed.